Frameworks/CoreText/DWriteWrapper.mm

```diff
@@ -527,19 +530,21 @@ HRESULT STDMETHODCALLTYPE GetPixelsPerDip(_In_opt_ void* clientDrawingContext, _
             i++;
         }
 
-        prevYPosForDraw = yPos;
+        if ([runs objectAtIndex:0] && static_cast<_CTRun*>([runs objectAtIndex:0])->_dwriteGlyphRun.glyphCount != 0) {
```

Comment:
sounds good, just wanted to know what was up

---
In reply to: [84180915](https://github.com/Microsoft/WinObjC/pull/1132#discussion_r84180915) [](ancestors = 84180915)